configman/def_sources/for_argparse.py

```diff
@@ -573,7 +575,7 @@ def parse_args(self, args=None, namespace=None):
             values_source_list=self.value_source_list,
             argv_source=args,
             app_name=self.prog,
-            app_version=self.version,
+#            app_version=self.version,
```

Comment:
My bad--I didn't read the code well. The version argument for `__init__` got deprecated, so that's gone after Python 3.2.

My vote is to change `ArgumentParser.__init__` to pop version off the kwargs using a default of `None` and save it as an attribute.